package.json

```diff
@@ -400,7 +400,7 @@
       "prettier --write"
     ],
     "+(frontend|enterprise/frontend)/**/*.{js,jsx,ts,tsx}": [
```

Comment:
This would only run on the code file changes, so if we remove a class from a css file and don't touch the source files it won't pick it up (in the precommit, it will in the CI).
The alternative would be to lint ALL source files when a css file changes, but it's going to make the precommit too long and from my experience people just use no-verify if it's longer than ~30 seconds

Reply:
I think that's ok for now. If this is able to handle cases other than removing a class then I think it's a good start